web/src/containers/activity/CostAllocate.test.js

```diff
@@ -33,48 +33,44 @@ describe('the CostAllocate component', () => {
 
   test('updates activity when text is changed', () => {
     const component = shallow(<CostAllocate {...props} />);
+    const mockedEvent = 
```

Comment:
The linter picked this up. I laughed when I saw this one because when I've done the same thing in the past, it's broken tests because I end up assigning something to this variable and then the tests don't work. Thankfully no such problem with this one! 😆

```suggestion
```